languages/python/sqlalchemy-oso/sqlalchemy_oso/oso.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, sqlalchemy_base):
         self.base = sqlalchemy_base
         self._roles_enabled = False
 
-    def enable_roles(self, user_model, session_maker):
+    def enable_roles(self, user_model, resource_id_column_type, session_maker):
```

Comment:
Can we add documentation of this new parameter (and the others, I know they're missing) to this doc string?